src/main/webapp/resources/js/pages/projects/project-analyses/components/ProjectAnalysesPage.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { Typography } from "antd";
+const { Title } = Typography;
+
+import { AnalysesTable } from "../../../../components/AnalysesTable/AnalysesTable";
+import { PagedTableProvider } from "../../../../components/ant.design/PagedTable";
+import { setBaseUrl } from "../../../../utilities/url-utilities";
+import { useGetProjectDetailsQuery } from "../../../../apis/projects/project";
+
```

Comment:
JSDoc

Reply:
Updated in 56cff88a0db10c4d82c97b83f21fe3c1fd5e2deb